### DISCUSSION TITLE
"Channel" missing in streams.json

### DISCUSSION THREAD

**Discussion by IPTVmanreal** [iptv-org/community#571] [Q&A] [answered]
The JSON's channel portion is entirely blank, causing any attempt to list channels invalid

![image](https://github.com/iptv-org/api/assets/99894372/e5ef640b-f8e4-4d51-8925-c6d87bc6a9aa)

**Comment by PopeyeTheSai10r** [comment]
Are you using `https://iptv-org.github.io/api/channels.json` ?

**Comment by IPTVmanreal** [comment]
No, I'm using https://iptv-org.github.io/api/streams.json

**Comment by PopeyeTheSai10r** [comment]
Working on my end. Try different browser or tool

![streams](https://github.com/iptv-org/api/assets/107763146/4d7b17f3-a043-4651-97e8-61f46a04402b)

**Comment by IPTVmanreal** [comment]
Your screenshot is also lacking the channel value
![image](https://github.com/iptv-org/api/assets/99894372/014b1b9c-a72d-43c7-90b4-3d863693ab7d)

**Comment by PopeyeTheSai10r** [comment]
Oh, I see. From your first screenshot, I though you were not able to get any response at all.

@freearhey any idea why the channel info isn't populating?

**Comment by PopeyeTheSai10r** [comment, marked as answer]
Upon further inspection, I noticed that not all the results has an empty channel value but some do. Those with no channel information are returned first. The information is based on what's the iptv repo the has for those channels.

If the tvg-id is left blank (""), it will return `""` when using the API.

Example: https://github.com/iptv-org/iptv/blob/ff68858512e18b2ff65fde11826d36ad62a5cab7/streams/am_ucom.m3u#L40
